core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java

```diff
@@ -265,6 +266,25 @@ public Component getStyledName(MapNameStyle style) {
         .collect(StreamUtils.toImmutableList());
   }
 
+  private Component parseGamemode(Element root) throws InvalidXMLException {
+    Component gamemode = XMLUtils.parseFormattedText(Node.fromLastChildOrAttr(root, "game"));
+
+    for (Element blitzTitle : XMLUtils.flattenElements(root, "blitz", "title")) {
+      if (this.getProto().isNoOlderThan(MapProtos.REMOVE_BLITZ_TITLE)) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is no longer supported, use <map game=\"...\"> or <game>",
+            blitzTitle);
+      } else if (gamemode != null && blitzTitle != null) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is overriden by <map game=\"...\"> or <game>", blitzTitle);
+      } else if (gamemode == null) {
+        gamemode = XMLUtils.parseFormattedText(Node.fromNullable(blitzTitle));
+      }
+    }
+
+    return gamemode;
+  }
```

Comment:
I believe this should have behavior identical to original for maps in the prior protos:

```suggestion
    Component gamemode = XMLUtils.parseFormattedText(root, "game");
    if (gamemode != null) return gamemode;

    for (Element title : XMLUtils.flattenElements(root, "blitz", "title")) {
      if (this.getProto().isNoOlderThan(MapProtos.REMOVE_BLITZ_TITLE)) {
        throw new InvalidXMLException(
            "<title> inside <blitz> is no longer supported, use <map game=\"...\"> or <game>",
            title);
      }
      return XMLUtils.parseFormattedText(new Node(title));
    }
  }
```
for reference (<https://github.com/OvercastNetwork/ProjectAres/blob/master/PGM/src/main/java/tc/oc/pgm/modules/InfoModule.java#L131>)

Reply:
Needed a return at the end, but yes.